### PR TITLE
chore: Add Go 1.19 image & use Kafka 1.9

### PR DESCRIPTION
**Pull request by minitauros**

### CODE REVIEW

File: alpine-golang/alpine3.16-golang1.19/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM golang:1.18-alpine3.16
```
Comment (Slijkhuis) [inline alpine-golang/alpine3.16-golang1.19/Dockerfile]:
```suggestion
FROM golang:1.19-alpine3.16
```

File: alpine-golang/alpine3.16-golang1.18/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18-alpine3.16
+FROM golang:1.19-alpine3.16
```
Comment (Slijkhuis) [inline alpine-golang/alpine3.16-golang1.18/Dockerfile]:
```suggestion
FROM golang:1.18-alpine3.16
```

File: Taskfile.yml [hunk 3]
```diff
@@ -2,8 +2,8 @@ version: '3'
 
 vars:
   ALPVER: 3.16
-  GOVER: 1.18
-  KAFKAVER: 1.6.2
+  GOVER: 1.19
+  KAFKAVER: 1.9.2
```
Comment (Slijkhuis) [inline Taskfile.yml:6]: The Kafka version is quite a bump right? Will that go well?

File: alpine-golang/alpine3.16-golang1.19/Dockerfile (hunk 1 above)
Comment (minitauros) [reply alpine-golang/alpine3.16-golang1.19/Dockerfile]: Wow I am an idiot haha 😆

File: Taskfile.yml (hunk 3 above)
Comment (minitauros) [reply Taskfile.yml:6]: I have no idea, but we don't use those images anymore, anyway.

My personal preference is usually to just try to roll the last version and revert if it doesn't work, vs stick to one and hope that there are no vulnerabilities. Also hoping for improvements in newer versions, and no breaking changes (aka proper semversioning). I mean, there can also be vulnerabilities in newer versions, of course, but well, that's worth the trade off to me.

I don't mind keeping at 1.6.2. Whatever you like! Let me know.

Comment (Slijkhuis) [reply Taskfile.yml:6]: Let's upgrade indeed.

**Review by Slijkhuis — APPROVED** [review]

**Pull request merged**